### PR TITLE
[nwd-nnw] adding works

### PR DESCRIPTION
**Pull request by Crazyayane07**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @Crazyayane07

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,63 @@
 #pragma once
 
-int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+int NWD(int lhs, int rhs) 
+{
+    int minValue = abs(lhs) >= abs(rhs) ? abs(rhs) : abs(lhs);
+	int maxValue = abs(lhs) >= abs(rhs) ? abs(lhs) : abs(rhs);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,63 @@
...
+
+	if (minValue > 1)
+	{
+		minValue = 1;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:10]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,63 @@
...
+	}
+
+	if (rhs == lhs) 
+	{
+		return lhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:15]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 4]
```diff
@@ -1,11 +1,63 @@
...
+
+	int result = minValue;
+	for (int i = minValue; i <= maxValue; i++)
+	{
+		if (i == 0) 
+		{
+			continue;
+		}
+
+		if (lhs%i == 0 && rhs%i == 0)
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:26]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 5]
```diff
@@ -1,11 +1,63 @@
...
+	{
+		return lhs;
+	}
+
+	int result = minValue;
+	for (int i = minValue; i <= maxValue; i++)
+	{
+		if (i == 0) 
+		{
+			continue;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:23]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 6]
```diff
@@ -1,11 +1,63 @@
...
+		}
+
+		if (lhs%i == 0 && rhs%i == 0) 
+		{
+			result = i;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:28]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 7]
```diff
@@ -1,11 +1,63 @@
...
+	{
+		if (result % lhs == 0 && result % rhs == 0)
+		{
+			foundNWW = true;
+		}
+		else 
+		{
+			result += multiplayer;
+		}
+	}
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:60]: Wrong indentation

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**